documentation.lisp

```diff
@@ -717,20 +736,45 @@ See STATUS")
   (function email
     "Returns the primary contact email address for the instance.
 
+See INSTANCE")
+
+  (function short-description
+    "Returns a short description of the object.
+
 See INSTANCE")
 
   (function version
     "Returns the instance's Mastodon version.
 
+See INSTANCE")
+
+  (function languages
+    "A list of ISO-6391 language codes that the instance advertises.
+
+See INSTANCE")
+
+  (function registrations
+    "Returns if is possible to register an user with this instance.
+
+See INSTANCE")
+
+  (function approval-required
+    "Returns if approval from moderator is required to register to this instance.
+
 See INSTANCE")
 
   (function urls
     "Returns a list of URLs for the streaming API.
 
 See INSTANCE")
 
-  (function languages
-    "A list of ISO-6391 language codes that the instance advertises.
+  (function stats
+    "Statistics about this instance (hashmap with legal keys: 'user-count', 'status-count', 'domain-count'.
```

Comment:
You seem to be missing closing parens in several of these docstrings.

It's also not clear whether user-count etc. are strings or symbols when you encase them in single quotes. Use double quotes for strings and SCREAM-CASE for symbols to improve clarity.